conf/references/mlss_conf.xml

```diff
@@ -57,94 +62,194 @@
+      <genome name="canis_lupus_familiaris"/>
     </collection>
 
     <collection name="mammalia">
-      <base_collection name="shared"/>
-      <!-- Outgroups -->
-      <genome name="ciona_intestinalis"/>
-      <genome name="dasypus_novemcinctus"/>
-      <genome name="xenopus_tropicalis"/>
-      <genome name="lepisosteus_oculatus"/>
-      <genome name="oryzias_latipes"/>
+      <base_collection name="shared_vertebrates"/>
+      <!-- Outgroups in shared vertebrates-->
       <!-- Prototheria -->
       <genome name="ornithorhynchus_anatinus"/>
       <!-- Metatheria -->
       <genome name="monodelphis_domestica"/>
-      <genome name="phascolarctos_cinereus"/>
+      <genome name="sarcophilus_harrisii"/>
+      <!-- Proboscidea -->
+      <genome name="loxodonta_africana"/>
+      <!-- 	Cingulata -->
+      <genome name="dasypus_novemcinctus"/>
+      <!-- Carnivora -->
+      <genome name="canis_lupus_familiaris"/>
+      <genome name="felis_catus"/>
+      <!-- Equidae -->
+      <genome name="equus_caballus"/>
+      <!-- Bats -->
+      <genome name="artibeus_jamaicensis"/>
+      <genome name="phyllostomus_discolor"/>
       <!-- Artiodactyla -->
-      <genome name="camelus_dromedarius"/>
+      <genome name="camelus_ferus"/>
       <genome name="bos_taurus"/>
       <genome name="ovis_aries_rambouillet"/>
       <genome name="capra_hircus"/>
       <genome name="sus_scrofa"/>
-      <genome name="delphinapterus_leucas"/>
-      <!-- Equidae -->
-      <genome name="equus_caballus"/>
-      <!-- Bats -->
-      <genome name="myotis_lucifugus"/>
-      <genome name="pteropus_vampyrus"/>
-      <!-- Shrew -->
-      <genome name="sorex_araneus"/>
+      <genome name="balaenoptera_musculus"/>
+      <!-- Primates -->
+      <genome name="microcebus_murinus"/>
+      <genome name="carlito_syrichta"/>
+      <genome name="macaca_mulatta"/>
+      <genome name="pan_troglodytes"/>
       <!-- Lagomorph -->
-      <genome name="oryctolagus_cuniculus"/>
+      <genome name="Oryctolagus_cuniculus"/>
       <!-- Rodents -->
       <genome name="mus_musculus"/>
       <genome name="rattus_norvegicus"/>
       <genome name="sciurus_vulgaris"/>
-      <!-- Primates -->
-      <genome name="macaca_mulatta"/>
-      <genome name="pan_troglodytes"/>
-      <genome name="callithrix_jacchus"/>
-      <genome name="prolemur_simus"/>
-      <!-- Carnivora -->
-      <genome name="canis_lupus_familiaris"/>
-      <genome name="felis_catus"/>
-      <genome name="suricata_suricatta"/>
-      <!-- Afrotheria -->
-      <genome name="echinops_telfairi"/>
+      <genome name="chinchilla_lanigera"/>
+      <genome name="dipodomys_ordii"/>
+      <genome name="jaculus_jaculus"/>
+      <genome name="peromyscus_maniculatus_bairdii"/>
+      <genome name="Nannospalax_galili"/>
+    </collection>
+
+    <collection name="actinopterygii">
+      <base_collection name="shared_vertebrates"/>
+      <!-- Outgroups -->
+      <genome name="ciona_intestinalis"/>
```

Comment:
```suggestion
```
As before

Reply:
cannot be removed because not in shared vertebrates